package-lock.json

```diff
@@ -21813,61 +21813,14 @@
       }
     },
     "react-native-onyx": {
-      "version": "git+https://github.com/Expensify/react-native-onyx.git#3635fc34f00c8c8f97500b97fd42251ab4d22ab5",
-      "from": "git+https://github.com/Expensify/react-native-onyx.git#3635fc34f00c8c8f97500b97fd42251ab4d22ab5",
+      "version": "git+https://github.com/Expensify/react-native-onyx.git#30c5e731467f8324593165871bedaebf16875c58",
+      "from": "git+https://github.com/Expensify/react-native-onyx.git#30c5e731467f8324593165871bedaebf16875c58",
       "requires": {
         "@react-native-community/async-storage": "^1.12.1",
-        "expensify-common": "git+https://github.com/Expensify/expensify-common.git#3d8fc7500ddd24cd4a543e6e160d4f1ad97cc145",
+        "expensify-common": "git+https://github.com/Expensify/expensify-common.git#758b46bc0b50320a006c7892faec56c2d3952135",
         "lodash": "4.17.21",
         "react": "^16.13.1",
         "underscore": "^1.11.0"
-      },
-      "dependencies": {
```

Comment:
I'm not sure why this got removed, maybe because it's no longer a sub-dependency of onyx and included as a first level dependency? cc @tgolen any idea?

Reply:
Looks like it's a direct dependency with the same commit number, I imagine that's why it doesn't need to be "duplicated" here. I'll let Tim confirm though.